File: intranet-angular.Server/Response/ConfiguracaoGrupoDeSlideResponse.cs
using intranet_angular.Server.Enuns;

namespace intranet_angular.Server.Response
{
    public class ConfiguracaoGrupoDeSlideResponse
    {
        public int Id { get; set; }
        public int PaginaId { get; set; }
        public int Posicao { get; set; }
        public TipoMidiaEnum TipoDeMidia { get; set; }
    }
}

File: intranet-angular.client/src/response/configuracaoGrupoDeSlideResponse.ts
import { TipoMidiaEnum } from "../app/enum/tipo-midia.enum";

export interface ConfiguracaoGrupoDeSlideResponse {
  id: number;
  paginaId: number;
  posicao: number;
  tipoDeMidia: TipoMidiaEnum;
}
